<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Value</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Result">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>520</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="6">
      <widget class="QLCDNumber" name="lcd07"/>
     </item>
     <item row="2" column="7">
      <widget class="QLCDNumber" name="lcd28"/>
     </item>
     <item row="2" column="0">
      <widget class="QLCDNumber" name="lcd21"/>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="lcd10"/>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="lcd24"/>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcd01"/>
     </item>
     <item row="1" column="7">
      <widget class="QLCDNumber" name="lcd18"/>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcd23"/>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="lcd14"/>
     </item>
     <item row="0" column="5">
      <widget class="QLCDNumber" name="lcd06"/>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd00"/>
     </item>
     <item row="0" column="7">
      <widget class="QLCDNumber" name="lcd08"/>
     </item>
     <item row="2" column="6">
      <widget class="QLCDNumber" name="lcd27"/>
     </item>
     <item row="1" column="6">
      <widget class="QLCDNumber" name="lcd17"/>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcd03"/>
     </item>
     <item row="2" column="5">
      <widget class="QLCDNumber" name="lcd26"/>
     </item>
     <item row="1" column="4">
      <widget class="QLCDNumber" name="lcd15"/>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcd22"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcd11"/>
     </item>
     <item row="2" column="4">
      <widget class="QLCDNumber" name="lcd25"/>
     </item>
     <item row="1" column="5">
      <widget class="QLCDNumber" name="lcd16"/>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcd12"/>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcd04"/>
     </item>
     <item row="0" column="4">
      <widget class="QLCDNumber" name="lcd05"/>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="lcd33"/>
     </item>
     <item row="3" column="3">
      <widget class="QLCDNumber" name="lcd34"/>
     </item>
     <item row="3" column="4">
      <widget class="QLCDNumber" name="lcd35"/>
     </item>
     <item row="3" column="5">
      <widget class="QLCDNumber" name="lcd36"/>
     </item>
     <item row="3" column="6">
      <widget class="QLCDNumber" name="lcd37"/>
     </item>
     <item row="3" column="7">
      <widget class="QLCDNumber" name="lcd38"/>
     </item>
     <item row="3" column="0">
      <widget class="QLCDNumber" name="lcd31"/>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcd32"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CHANNEL 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CHANNEL 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CHANNEL 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CHANNEL 4</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>431</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="6">
      <widget class="QLCDNumber" name="lcd07_3"/>
     </item>
     <item row="2" column="7">
      <widget class="QLCDNumber" name="lcd28_3"/>
     </item>
     <item row="2" column="0">
      <widget class="QLCDNumber" name="lcd21_3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="lcd10_3"/>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="lcd24_3"/>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcd01_3"/>
     </item>
     <item row="1" column="7">
      <widget class="QLCDNumber" name="lcd18_3"/>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcd23_3"/>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="lcd14_3"/>
     </item>
     <item row="0" column="5">
      <widget class="QLCDNumber" name="lcd06_3"/>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd00_3"/>
     </item>
     <item row="0" column="7">
      <widget class="QLCDNumber" name="lcd08_3"/>
     </item>
     <item row="2" column="6">
      <widget class="QLCDNumber" name="lcd27_3"/>
     </item>
     <item row="1" column="6">
      <widget class="QLCDNumber" name="lcd17_3"/>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcd03_3"/>
     </item>
     <item row="2" column="5">
      <widget class="QLCDNumber" name="lcd26_3"/>
     </item>
     <item row="1" column="4">
      <widget class="QLCDNumber" name="lcd15_3"/>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcd22_3"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcd11_3"/>
     </item>
     <item row="2" column="4">
      <widget class="QLCDNumber" name="lcd25_3"/>
     </item>
     <item row="1" column="5">
      <widget class="QLCDNumber" name="lcd16_3"/>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcd12_3"/>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcd04_3"/>
     </item>
     <item row="0" column="4">
      <widget class="QLCDNumber" name="lcd05_3"/>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="lcd33_3"/>
     </item>
     <item row="3" column="3">
      <widget class="QLCDNumber" name="lcd34_3"/>
     </item>
     <item row="3" column="4">
      <widget class="QLCDNumber" name="lcd35_3"/>
     </item>
     <item row="3" column="5">
      <widget class="QLCDNumber" name="lcd36_3"/>
     </item>
     <item row="3" column="6">
      <widget class="QLCDNumber" name="lcd37_3"/>
     </item>
     <item row="3" column="7">
      <widget class="QLCDNumber" name="lcd38_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLCDNumber" name="lcd31_3"/>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcd32_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="4">
      <widget class="QLCDNumber" name="lcd25_4"/>
     </item>
     <item row="0" column="7">
      <widget class="QLCDNumber" name="lcd08_4"/>
     </item>
     <item row="1" column="7">
      <widget class="QLCDNumber" name="lcd18_4"/>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd00_4"/>
     </item>
     <item row="1" column="5">
      <widget class="QLCDNumber" name="lcd16_4"/>
     </item>
     <item row="3" column="5">
      <widget class="QLCDNumber" name="lcd36_4"/>
     </item>
     <item row="0" column="5">
      <widget class="QLCDNumber" name="lcd06_4"/>
     </item>
     <item row="2" column="7">
      <widget class="QLCDNumber" name="lcd28_4"/>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcd12_4"/>
     </item>
     <item row="1" column="6">
      <widget class="QLCDNumber" name="lcd17_4"/>
     </item>
     <item row="0" column="6">
      <widget class="QLCDNumber" name="lcd07_4"/>
     </item>
     <item row="2" column="5">
      <widget class="QLCDNumber" name="lcd26_4"/>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcd04_4"/>
     </item>
     <item row="3" column="3">
      <widget class="QLCDNumber" name="lcd34_4"/>
     </item>
     <item row="1" column="4">
      <widget class="QLCDNumber" name="lcd15_4"/>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="lcd10_4"/>
     </item>
     <item row="2" column="0">
      <widget class="QLCDNumber" name="lcd21_2"/>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcd23_4"/>
     </item>
     <item row="2" column="6">
      <widget class="QLCDNumber" name="lcd27_4"/>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcd22_4"/>
     </item>
     <item row="0" column="4">
      <widget class="QLCDNumber" name="lcd05_4"/>
     </item>
     <item row="3" column="4">
      <widget class="QLCDNumber" name="lcd35_4"/>
     </item>
     <item row="3" column="0">
      <widget class="QLCDNumber" name="lcd31_4"/>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="lcd33_4"/>
     </item>
     <item row="3" column="7">
      <widget class="QLCDNumber" name="lcd38_4"/>
     </item>
     <item row="3" column="6">
      <widget class="QLCDNumber" name="lcd37_4"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcd11_4"/>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcd32_4"/>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcd01_4"/>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="lcd14_4"/>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcd03_4"/>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="lcd24_4"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>431</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="6">
      <widget class="QLCDNumber" name="lcd07_5"/>
     </item>
     <item row="2" column="7">
      <widget class="QLCDNumber" name="lcd28_5"/>
     </item>
     <item row="2" column="0">
      <widget class="QLCDNumber" name="lcd21_4"/>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="lcd10_5"/>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="lcd24_5"/>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcd01_5"/>
     </item>
     <item row="1" column="7">
      <widget class="QLCDNumber" name="lcd18_5"/>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcd23_5"/>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="lcd14_5"/>
     </item>
     <item row="0" column="5">
      <widget class="QLCDNumber" name="lcd06_5"/>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd00_5"/>
     </item>
     <item row="0" column="7">
      <widget class="QLCDNumber" name="lcd08_5"/>
     </item>
     <item row="2" column="6">
      <widget class="QLCDNumber" name="lcd27_5"/>
     </item>
     <item row="1" column="6">
      <widget class="QLCDNumber" name="lcd17_5"/>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcd03_5"/>
     </item>
     <item row="2" column="5">
      <widget class="QLCDNumber" name="lcd26_5"/>
     </item>
     <item row="1" column="4">
      <widget class="QLCDNumber" name="lcd15_5"/>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcd22_5"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcd11_5"/>
     </item>
     <item row="2" column="4">
      <widget class="QLCDNumber" name="lcd25_5"/>
     </item>
     <item row="1" column="5">
      <widget class="QLCDNumber" name="lcd16_5"/>
     </item>
     <item row="1" column="2">
      <widget class="QLCDNumber" name="lcd12_5"/>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcd04_5"/>
     </item>
     <item row="0" column="4">
      <widget class="QLCDNumber" name="lcd05_5"/>
     </item>
     <item row="3" column="2">
      <widget class="QLCDNumber" name="lcd33_5"/>
     </item>
     <item row="3" column="3">
      <widget class="QLCDNumber" name="lcd34_5"/>
     </item>
     <item row="3" column="4">
      <widget class="QLCDNumber" name="lcd35_5"/>
     </item>
     <item row="3" column="5">
      <widget class="QLCDNumber" name="lcd36_5"/>
     </item>
     <item row="3" column="6">
      <widget class="QLCDNumber" name="lcd37_5"/>
     </item>
     <item row="3" column="7">
      <widget class="QLCDNumber" name="lcd38_5"/>
     </item>
     <item row="3" column="0">
      <widget class="QLCDNumber" name="lcd31_5"/>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcd32_5"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="test">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>530</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="res">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>524</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <zorder>Result</zorder>
   <zorder>Button</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>gridLayoutWidget_2</zorder>
   <zorder>gridLayoutWidget_3</zorder>
   <zorder>gridLayoutWidget_4</zorder>
   <zorder>test</zorder>
   <zorder>res</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
